conf/test.config

```diff
@@ -10,24 +10,27 @@
 ----------------------------------------------------------------------------------------
 */
 
+process {
+    resourceLimits = [
+        cpus: 4,
+        memory: '15.GB',
+        time: '1.h'
```

Comment:
1h limit might be problematic for slow downloads (e.g. amrfinderplus database)? All test profiles got updated with the same values, are they mandatory?